xrpl/models/base_model.py

```diff
@@ -176,6 +177,10 @@ def _from_dict_single_param(
             # expected an object, received the correct object
             return param_value
 
+        if get_origin(param_type) == Literal:
+            if param_value in get_args(param_type):
+                return param_value
```

Comment:
Is this saying that if the type is something like `Literal[param_type]` and `param_value` is one of the literals there, we're good to go? 

(Maybe worth a comment like `# To parse Literal[param_value]`